### PR TITLE
Skip failed views and functions

### PR DESCRIPTION
**Pull request by ondrajodas**
Fix error:
```
SQL error: SQL compilation error: error line 19 at position 41 Session variable '$PERS_COSTS_SNAPSHOT_DATE' does not exist, SQL state 02000 in SQLPrepare" while executing query "CREATE OR REPLACE VIEW
```

### CODE REVIEW

File: src/Migrate.php [hunk 1]
```diff
@@ -624,7 +625,17 @@ function (array $v) use ($database, $schemaName, $tableName) {
                     Helper::quoteIdentifier($view['schema_name'])
                 ));
 
-                $this->destinationConnection->query($view['text']);
+                try {
+                    $this->destinationConnection->query($view['text']);
+                } catch (Throwable $e) {
+                    $this->logger->warning(sprintf(
```
Comment (pivnicek) [inline src/Migrate.php]: We had a truncation problem dumping everything to stderr, (the event got truncated), so I switched the others to `info` instead.  Makes it harder to parse, but at least we catch them all

**Review by pivnicek — DISMISSED** [review]

**Comment by pivnicek** [comment]
I approved, because I don't want to block merging, but pls switch to `info` messages

File: src/Migrate.php [hunk 2]
```diff
@@ -624,7 +625,17 @@ function (array $v) use ($database, $schemaName, $tableName) {
                     Helper::quoteIdentifier($view['schema_name'])
                 ));
 
-                $this->destinationConnection->query($view['text']);
+                try {
+                    $this->destinationConnection->query($view['text']);
+                } catch (Throwable $e) {
+                    $this->logger->info(sprintf(
+                        'Skip creating view %s.%s.%s. Error: "%s".',
```
Comment (pivnicek) [inline src/Migrate.php]: ahhh, but please keep `Warning:` text so we can gather them:)

Comment (ondrajodas) [reply src/Migrate.php]: okok :)

**Review by pivnicek — APPROVED** [review]

**Pull request merged**